README.md

```diff
@@ -8,30 +8,52 @@ This repo contains a base Docker image for use by [Meteor](https://www.meteor.co
 
 ## Quickstart
 
+### Step #1 - Bootstrap Dockerfile from template
+
 Copy `example/default.dockerfile` (or `example/app-with-native-dependencies.dockerfile` if your app has native dependencies that require compilation such as `bcrypt`, or if your app is using a version of Meteor older than 1.8.1) into the root of your project and rename it `Dockerfile`. This file assumes that your Meteor app is one level down from the root in a folder named `app`; either move your app there, or edit `Dockerfile` to point to your desired path (or the root of your project). Leave `Dockerfile` at the root.
 
-Edit the `Dockerfile` you copied into your project, changing the first line so that the numbers at the end match the version of Meteor of your project. For example:
+### Step #2 - Ensure the Meteor version is the one you want
+
+Edit the `Dockerfile` you copied into your project, changing the first line so that the numbers at the end match the version of Meteor of your project.
+
+For example if you project is running under Meteor 2.2:
 
 ```Dockerfile
 FROM geoffreybooth/meteor-base:2.2
 ```
 
-if your project is running under Meteor 2.2. See your app’s `.meteor/release` file to get its Meteor release version. This version must match an available tag from [geoffreybooth/meteor-base](https://hub.docker.com/r/geoffreybooth/meteor-base/tags).
+See your app’s `.meteor/release` file to get its Meteor release version. This version must match an available tag from [geoffreybooth/meteor-base](https://hub.docker.com/r/geoffreybooth/meteor-base/tags).
 
 If necessary, update version in the `FROM node` line to use the Node version appropriate for your release of Meteor. From your application folder, you can get this version via the following command:
 
 ```bash
 docker run --rm geoffreybooth/meteor-base:$(cat ./.meteor/release | cut -c8-99) meteor node --version | cut -c2-99 | grep -o "[0-9\.]*"
 ```
 
-Also copy in `example/.dockerignore` and `example/docker-compose.yml` to your project’s root. Then, from the root of your project:
+### Step #3 - Setup your .dockerignore accordingly to speed up builds
+
+Also copy the `example/.dockerignore` file to the root of your repository so that heavy non-needed files are not passed to the Docker context, so that your build is faster (for more, see the [`.dockerignore` documentation](https://docs.docker.com/engine/reference/builder/#dockerignore-file).
+
+### Step #4 - Build & Profit 🎉
+
+Congratulations, you can know build the Docker image for your Meteor app in an easy and efficient way 🙂
+
+```bash
+docker build -t you-awesome-name/your-meteor-app-name .
+```
+
+### Step #5 (Optional) - Use Docker Compose to ease your workflow
+
```

Comment:
```suggestion
```

I would cut all this. It’s simpler to give only one way to do something rather than two, and the `docker-compose` approach is more powerful and just as easy.